tests/phpunit/data/structuredmacro-pagetree-output.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<xml xmlns:ac="sample_namespace" xmlns:ri="sample_second_namespace">
+	<div class="ac-pagetree">{{PageTree|reverse=false|sort=natural|startDepth=3|excerpt=true|searchBox=true|expandCollapseAll=true|content-title=ABC:SomeLinkedPage/Testpage}}</div>
+
+	<div class="ac-pagetree">{{PageTree|space-key=DEVOPS|content-title=ABC:Main Page}}</div>
+
+	<div class="ac-pagetree">{{PageTree|space-key=DEVOPS|content-title=ABC:SomeLinkedPage/Testpage}}</div>
+
+	<div class="ac-pagetree">{{PageTree|space-key=DEVOPS|content-title=ABC:SomeLinkedPage}}</div>
+
+	<div class="ac-pagetree" data-params="{&quot;space-key&quot;:&quot;DEVOPS&quot;}">
+[[Category:Broken_macro/pagetree]]
```

Comment:
Can't we just replace it with
```
{{PageTree|space-key=DEVOPS}}[[Category:Broken_macro/pagetree]]
```

* If no `content-title` is set we could fall back to `{{FULLPAGENAME}}`
* If no `space-key` is set we could fall back to `{{NAMESPACE}}`

Reply:
Done but SubpageTree does not work for namespace